opsramp/devmgmt.py

```diff
@@ -41,3 +41,46 @@ def run(self, uuid):
 
     def delete(self, uuid):
         return self.api.delete('%s' % uuid)
+
+
+class Discovery(ApiWrapper):
+    def __init__(self, parent):
+        super(Discovery, self).__init__(parent.api, 'policies/discovery')
+
+    def search(self, pattern=''):
+        return self.api.get('/search?name=%s' % pattern)
+
+    def get(self, discoveryProfileId=''):
```

Comment:
Don't actually need this function because there's a get(uuid) in the baseclass ApiWrapper that gets the subobject with the specified uuid. Most of the "collection of objects" API endpoints work that way, which is why I put get(uuid) in the baseclass. You only need to write a specific get if you want to do something different.